app/parse.py

```diff
@@ -8,8 +12,59 @@ class Quote:
     tags: list[str]
 
 
+BASE_URL = "https://quotes.toscrape.com/"
```

Comment:
place constant(s) into class/function or program start-point to not create redundant imports at the unexpected program places